cves/kernel/CVE-2017-15126.yml

```diff
@@ -286,8 +287,8 @@ ipc:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
```

Comment:
Fork is by definition IPC between a process and the kernel. You may have missread the question. It asks if the feature this vulnerablity affected involves IPC not if the vulnerability was caused by IPC